src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -400,14 +400,14 @@ protected ResponseEntity<Object> searchUsersByOrganisation(String organisationId
         showDeleted = getShowDeletedValue(showDeleted);
         returnRoles = getReturnRolesValue(returnRoles);
 
-        if (page != null) {
+        if (page != null && userIdentifier == null) {
```

Comment:
This is logic is wrong. If you ask for pagination and don't provide a user identifier you don't get pagination

Reply:
Hi @andywilkinshmcts apologies !! please correct in case am wrong , the pagination is required for multiple users , if only one user is requested through userIdentifier then applying a pagination is not required , thats why here the pagination logic is not applied for a single user